test/integration/pdf_generation_test.exs

```diff
@@ -394,6 +394,25 @@ defmodule ChromicPDF.PDFGenerationTest do
     end
   end
 
+  describe "timezone can be set thru print_to_pdf/2" do
+    setup do
+      start_supervised!({ChromicPDF, offline: false})
+      start_supervised!(TestServer.bandit(:http))
+
+      %{port: TestServer.port(:http)}
+    end
+
+    @tag :disable_logger
+    test "timezone can be set", %{port: port} do
+      input = {:url, "http://localhost:#{port}/timezone_echo"}
+
+      print_to_pdf(input, [set_timezone: %{timezoneId: "America/New_York"}], fn text ->
```

Comment:
Since `timezoneId` seems to be the only parameter of this, why not just simplify the api a bit: 

```suggestion
      print_to_pdf(input, [timezone: "America/New_York"], fn text ->
```

Reply:
Thank you for that feedback, @andreasknoepfle! I've made that change, and also updated the `CHANGELOG` (thank you for suggesting to look through history for an example).